python/ray/dashboard/modules/reporter/healthz_agent.py

```diff
@@ -52,6 +52,26 @@ async def health_check(self, req: Request) -> Response:
             content_type="application/text",
         )
 
+    @routes.get("/api/healthz")
+    async def unified_health(self, req: Request) -> Response:
```

Comment:
Suggest to:

- Always check the health of both (concurrently). That way we know if only raylet is unhealthy or GCS is too. As-is, you wouldn't know if GCS is healthy.
- Warning log if health check fails for either of them.